gensrc/script/vectorized/vectorized_functions.py

```diff
@@ -689,18 +689,18 @@
     [150200, 'array_concat', 'ARRAY_DECIMALV2', ['ARRAY_DECIMALV2', "..."], 'ArrayFunctions::array_concat_decimalv2'],
     [150201, 'array_concat', 'ARRAY_VARCHAR',   ['ARRAY_VARCHAR', "..."],   'ArrayFunctions::array_concat_varchar'],
 
-    [150210, 'array_overlap', 'BOOLEAN',      ['ARRAY_DATE', 'ARRAY_DATE'],        'ArrayFunctions::array_overlap_date'],
-    [150211, 'array_overlap', 'BOOLEAN',  ['ARRAY_DATETIME', 'ARRAY_DATETIME'],    'ArrayFunctions::array_overlap_datetime'],
-    [150212, 'array_overlap', 'BOOLEAN',   ['ARRAY_BOOLEAN', 'ARRAY_BOOLEAN'],     'ArrayFunctions::array_overlap_boolean'],
-    [150213, 'array_overlap', 'BOOLEAN',   ['ARRAY_TINYINT', 'ARRAY_TINYINT'],     'ArrayFunctions::array_overlap_tinyint'],
-    [150214, 'array_overlap', 'BOOLEAN',  ['ARRAY_SMALLINT', 'ARRAY_SMALLINT'],    'ArrayFunctions::array_overlap_smallint'],
-    [150215, 'array_overlap', 'BOOLEAN',       ['ARRAY_INT', 'ARRAY_INT'],         'ArrayFunctions::array_overlap_int'],
-    [150216, 'array_overlap', 'BOOLEAN',    ['ARRAY_BIGINT', 'ARRAY_BIGINT'],      'ArrayFunctions::array_overlap_bigint'],
-    [150217, 'array_overlap', 'BOOLEAN',  ['ARRAY_LARGEINT', 'ARRAY_LARGEINT'],    'ArrayFunctions::array_overlap_largeint'],
-    [150218, 'array_overlap', 'BOOLEAN',     ['ARRAY_FLOAT', 'ARRAY_FLOAT'],       'ArrayFunctions::array_overlap_float'],
-    [150219, 'array_overlap', 'BOOLEAN',    ['ARRAY_DOUBLE', 'ARRAY_DOUBLE'],      'ArrayFunctions::array_overlap_double'],
-    [150220, 'array_overlap', 'BOOLEAN', ['ARRAY_DECIMALV2', 'ARRAY_DECIMALV2'],   'ArrayFunctions::array_overlap_decimalv2'],
-    [150221, 'array_overlap', 'BOOLEAN',   ['ARRAY_VARCHAR', 'ARRAY_VARCHAR'],     'ArrayFunctions::array_overlap_varchar'],
+    [150210, 'arrays_overlap', 'BOOLEAN',      ['ARRAY_DATE', 'ARRAY_DATE'],        'ArrayFunctions::array_overlap_date'],
```

Comment:
```suggestion
    [150210, 'arrays_overlap', 'BOOLEAN',      ['ARRAY_DATE', 'ARRAY_DATE'],        'ArrayFunctions::arrays_overlap_date'],
```

Reply:
which are internal names, I think using  array_overlap is OK. otherwise， the change will be a lot.